extensions/twenty/src/services/TwentySDK.ts

```diff
@@ -0,0 +1,82 @@
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import { getPreferenceValues } from "@raycast/api";
+import { Api } from "../enum/api";
+import fetch from "node-fetch";
+import { getActiveDataModelsSchema } from "./zod/schema/dataModelSchema";
+import { dataModelsFieldsSchema } from "./zod/schema/recordFieldSchema";
+
+class TwentySDK {
+  private url!: string;
+  private token!: string;
+
+  constructor() {
+    const { token, url }: { token: string; url: string } = getPreferenceValues();
```

Comment:
Preferences are auto-generated and exposed using `Preferences` type so you don't need to add types yourself.
```suggestion
    const { token, url } = getPreferenceValues<Preferences>();
```